dll/win32/imm32/ime.c

```diff
@@ -297,7 +297,7 @@ PIMEDPI APIENTRY Imm32LoadImeDpi(HKL hKL, BOOL bLock)
     pImeDpiNew->hKL = hKL;
 
     lcid = LOWORD(hKL);
-    if (TranslateCharsetInfo((LPDWORD)(DWORD_PTR)lcid, &ci, TCI_SRCLOCALE))
+    if (TranslateCharsetInfo((LPDWORD)UlongToPtr(lcid), &ci, TCI_SRCLOCALE))
```

Comment:
The LPDWORD cast might be removed, if this doesn't cause MSVC nor GCC to warn or err.

Reply:
Removed.